### PR TITLE
Fix to the remote_debugger division by zero

### PR DESCRIPTION
**Pull request by Mek101**
Quick fix for: https://sonarcloud.io/project/issues?id=qarmin_godot&issues=AXDOROEvlmAOxIdHqVe6&open=AXDOROEvlmAOxIdHqVe6
The bandwith usage will be zero if the given buffer is empty

### CODE REVIEW

File: core/debugger/remote_debugger.cpp [hunk 1]
```diff
@@ -78,18 +78,22 @@ struct RemoteDebugger::NetworkProfiler {
...
-
-		int i = (p_pointer + p_buffer.size() - 1) % p_buffer.size();
-
-		while (i != p_pointer && p_buffer[i].packet_size > 0) {
-			if (p_buffer[i].timestamp < final_timestamp) {
-				return total_bandwidth;
+		
+		// No frames no usage.
+		if(p_buffer.size()) // Not zero
+		{
```
Comment (volzhs) [inline core/debugger/remote_debugger.cpp:84]: little style issue here
```suggestion
		if (p_buffer.size()) {
```

Comment (Mek101) [reply core/debugger/remote_debugger.cpp:84]: Where do I find the style guide/standard?

Comment (volzhs) [reply core/debugger/remote_debugger.cpp]: https://github.com/godotengine/godot/tree/master/misc/hooks
this will help.

Comment (Calinou) [reply core/debugger/remote_debugger.cpp]: See [Using clang-format locally](https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#using-clang-format-locally) as well.

**Pull request closed without merging**